recipes-connectivity/networkmanager/networkmanager-1.18.4/fix-compatibility-with-network-slave.patch

```diff
@@ -152,352 +152,6 @@
-+        Reapply can make the applied-connection different from the
-+        settings-connection, just like updating the settings-connection can make
-+        them different.
-+    -->
-+    <method name="Reapply">
-+      <arg name="connection" type="a{sa{sv}}" direction="in"/>
-+      <arg name="version_id" type="t" direction="in"/>
-+      <arg name="flags" type="u" direction="in"/>
-+      </method>
-+
-+    <!--
-+        GetAppliedConnection:
-+        @flags: Flags which would modify the behavior of the GetAppliedConnection call. There are no flags defined currently and the users should use the value of 0.
-+        @connection: The effective connection settings that the connection has currently applied.
-+        @version_id: The version-id of the currently applied connection. This can be specified during Reapply to avoid races where you first fetch the applied connection, modify it and try to reapply it. If the applied connection is modified in the meantime, the version_id gets incremented and Reapply will fail.
-+
-+        Get the currently applied connection on the device. This is a snapshot of
-+        the last activated connection on the device, that is the configuration
-+        that is currently applied on the device. Usually this is the same as
-+        GetSettings of the referenced settings connection. However, it can differ
-+        if the settings connection was subsequently modified or the applied
-+        connection was modified by Reapply. The applied connection is set when
-+        activating a device or when calling Reapply.
-+    -->
-+    <method name="GetAppliedConnection">
-+      <arg name="flags" type="u" direction="in"/>
-+      <arg name="connection" type="a{sa{sv}}" direction="out"/>
-+      <arg name="version_id" type="t" direction="out"/>
-+      </method>
-+
-+    <!--
-+        Disconnect:
-+
-+        Disconnects a device and prevents the device from automatically activating
-+        further connections without user intervention.
-+    -->
-+    <method name="Disconnect"/>
-+
-+    <!--
-+        Delete:
-+
-+        Deletes a software device from NetworkManager and removes the interface
-+        from the system. The method returns an error when called for a hardware
-+        device.
-+    -->
-+    <method name="Delete"/>
-+
-+    <!--
-+        StateChanged:
-+        @new_state: (<link linkend="NMDeviceState">NMDeviceState</link>) The new state of the device.
-+        @old_state: (<link linkend="NMDeviceState">NMDeviceState</link>) The previous state of the device.
-+        @reason: (<link linkend="NMDeviceStateReason">NMDeviceStateReason</link>) A reason for the state transition.
-+    -->
-+    <signal name="StateChanged">
-+      <arg name="new_state" type="u"/>
-+      <arg name="old_state" type="u"/>
-+      <arg name="reason" type="u"/>
-+    </signal>
-+
 +    <tp:enum name="NM_DEVICE_STATE" type="u">
 +      <tp:enumvalue suffix="UNKNOWN" value="0">
 +        <tp:docstring>
```

Comment:
Do all the `<tp:docstring>`s need to change to `<!-- -->`?

(It built for me if I left the `<interface>` ahead of all the `<tp:enum>`, which is a smaller diff)